### PR TITLE
Update devices.py to make it compatible with latest SONiC image

### PR DESCRIPTION
**Pull request by bingwang-ms**
### Description of PR

Summary:
1. Recent SONiC image added dependent-startup service to docker images, and start.sh is renamed to start. These two updates cause check_pmon_daemon_status return false;
2. The latest image introduces command 'show feature status' to get feature list. This commit update the command used in get_feature_status, and keep compatible with pre versions and the same time.

### Type of change

- [x] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

#### What is the motivation for this PR?
This PR is to update devices.py to make it compatible with latest SONiC image.
#### How did you do it?
1. Update the ignore list in check_pmon_daemon_status;
2. Update the command used in get_feature_status.
#### How did you verify/test it?
Verified on Arista-7260.
#### Any platform specific information?
No.
#### Supported testbed topology if it's a new test case?
N/A
### Documentation 

N/A

### CODE REVIEW

**Comment by lguohan** [comment]
retest this please

File: tests/common/devices.py [hunk 1]
```diff
@@ -758,9 +758,13 @@ def get_feature_status(self):
...
-        command_output = self.shell('show features', module_ignore_errors=True)
-        if command_output['rc'] != 0:
-            return feature_status, False
+        command_list = ['show feature status', 'show features']
+        for cmd in command_list:
+            command_output = self.shell(cmd, module_ignore_errors=True)
+            if command_output['rc'] == 0:
+                break
+            else:
+                return feature_status, False
```
Comment (wangxin) [inline tests/common/devices.py:767]: If the first command failed, then the second command will not be tried. Is this correct?

File: tests/common/devices.py [hunk 2]
```diff
@@ -758,9 +758,13 @@ def get_feature_status(self):
...
         feature_status = {}
-        command_output = self.shell('show features', module_ignore_errors=True)
-        if command_output['rc'] != 0:
-            return feature_status, False
+        command_list = ['show feature status', 'show features']
+        for cmd in command_list:
+            command_output = self.shell(cmd, module_ignore_errors=True)
+            if command_output['rc'] == 0:
+                break
+            else:
```
Comment (neethajohn) [inline tests/common/devices.py:766]: This 'else' should pair with the 'for' construct not the 'if' construct

Comment (bingwang-ms) [reply tests/common/devices.py:766]: Thanks Neetha. It's a bug. I'll make an update.

File: tests/common/devices.py (hunk 1 above)
Comment (bingwang-ms) [reply tests/common/devices.py:767]: Thanks Xin. It's a bug. I'll make an update.

**Pull request merged**